doc/how_to_use.md

```diff
@@ -0,0 +1,131 @@
+# How to use Sōzu
+
+> Before to deep dive in the configuration part of the proxy. You should take a look at the [getting start](./getting_started.md) if you haven't done it yet.
+
+## Configuration file
+
+> The configuration file is a [.toml](https://github.com/toml-lang/toml)
+
+Sōzu configuration process involves 3 major sources of parameters:
+
+* The `global` section, which sets process-wide parameters.
+* The definition of the frontends like `https`, `http`, `tcp`.
+* The proxies sections under: `[applications]`.
+
+### Global parameters
+
+Parameters in the global section can be use to define the global settings shared by the master and workers (like the log level):
+
+* `command_socket` path to the unix socket command (see below for more infos)
+* `saved_state` path to file to dump the proxy state
+* `log_level` possible values are: `debug, trace, error, info`
+* `log_target` possible values are: `stdout, tcp or udp address`
+* `command_buffer_size` size of the buffer use by the master to process commands.
+* `worker_count` set the number of workers
+* `handle_process_affinity` bind workers to cpu cores.
+* `max_connections` maximum number of connections
+* `max_buffers` 
+* `buffer_size` size of requests buffer use by the workers
+
+*Example:*
+``` toml
+command_socket = "./command_folder/sock"
+saved_state = "./state.json"
+log_level = "info"
+log_target = "stdout"
+command_buffer_size = 16384
+worker_count = 2
+handle_process_affinity = false
+max_connections = 500
+max_buffers = 500
+buffer_size = 16384
+```
+
+### Frontends
```

Comment:
Please do not call those frontends, call them protocols, otherwise this will be very confusing when we explain the actual application frontends